src/test/resources/services-local.conf

```diff
@@ -19,9 +30,11 @@
 
 
 services {
-    example-frontend{
-      protocol = http
-      host = localhost
-      port = 9080
-    }
+           auth-login-api.port = 8585
```

Comment:
Is this used? It seems weird to have this as well as the auth-login-stub (especially with the same port value) - especially as I don't see a reference to auth-login-api in the PR